Content.Shared/Chat/ChatChannel.cs

```diff
@@ -19,7 +19,7 @@ public enum ChatChannel : ushort
         Whisper = 1 << 1,
```

Comment:
This beauty should be at some point replaced with protos, mostly (and ProtoId i guess). I'm not 100% sure if that should be done on that PR.

Reply:
flags combinations probably could be replaced by some simple helper methods than